packages/augur-ui/src/modules/create-market/actions/get-category-stats.ts

```diff
@@ -0,0 +1,16 @@
+import { AppState } from 'store';
+import { ThunkAction } from 'redux-thunk';
+import { augurSdk } from 'services/augursdk';
+import { POPULAR_CATEGORIES } from 'modules/common/constants';
+
+export const getCategoryStats = (
+  universe: string,
+  cb: Function = () => {}
+): ThunkAction<void, AppState, void, any> => async (
+  dispatch,
```

Comment:
Doesnt look you need the dispatch and getState stuff, since redux isnt involved.